packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -96,7 +96,9 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         'onFileSystem',
         'onCustomEditor',
         'onStartupFinished',
-        'onAuthenticationRequest'
+        'onAuthenticationRequest',
+        'onNotebook',
```

Comment:
Don't we need to add something to `plugin-activation-events.ts` to make this work?

Reply:
yeah i have to take a look at that. I was kind of confused by the onNotebook and onNotebookSerializer events.